internal/cron.go

```diff
@@ -53,6 +58,10 @@ func (s *cronsumer) ProduceBatch(messages []kafka.Message) error {
 	return s.consumer.kafkaProducer.ProduceBatch(messages)
 }
 
+func (s *cronsumer) GetMetricCollectors() []prometheus.Collector {
```

Comment:
Do we need to expose this function in `pkg/cronsumer.go` within `Cronsumer interface` ?

Reply:
Certainly. To gain access to MetricsCollector in konsumer